src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -97,19 +112,31 @@ export default class LimitedRestorationPanel extends Mixins(DateMixin) {
   private setCustomMonths (val) {
     this.customMonths = val === 'customMonths'
     if (val !== 'customMonths') {
+      // Clear text field when another radio button is selected
+      if (this.selectMonths !== 'customMonths') {
+        this.numberOfMonths = null
+        this.$refs.monthsRef.reset()
+      }
+      this.monthsValid(true)
       this.expiryChanged(this.addMonthsToDate(val, this.currentDate))
     } else {
       this.expiryChanged(this.addMonthsToDate(this.numberOfMonths, this.currentDate))
     }
   }
 }
-
 </script>
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
-
 .month-text {
   color: $gray7;
 }
+
+::v-deep .number-months-field{
```

Comment:
Add a comment explaining what this is doing (or why).

Also, need a space before `{`.